packages/flutter/test/material/page_test.dart

```diff
@@ -245,6 +245,19 @@ void main() {
     // Page 1 is now visible.
     expect(find.text('Page 1'), isOnstage);
     expect(find.text('Page 2'), isOnstage);
+
+    // The route widget position needs to track the finger position very exactly.
+    expect(tester.getTopLeft(find.text('Page 2')), const Point(400.0, 0.0));
+
+    await gesture.moveBy(const Offset(-200.0, 0.0));
+    await tester.pumpAndSettle();
```

Comment:
why pumpAndSettle rather than just pump?

Please see the docs for pumpAndSettle -- we should only use it when strictly necessary and should strongly prefer using explicit pumps normally.